Source/WebInspectorUI/UserInterface/Views/SpreadsheetRuleHeaderField.js

```diff
@@ -111,10 +113,11 @@ WI.SpreadsheetRuleHeaderField = class SpreadsheetRuleHeaderField extends WI.Obje
         if (document.activeElement === this._element)
             return;
 
-        if (this._delegate?.spreadsheetRuleHeaderFieldDidCommit) {
-            let changed = this._element.textContent !== this._valueBeforeEditing;
-            this._delegate.spreadsheetRuleHeaderFieldDidCommit(this, changed);
-        }
+        let changed = this._element.textContent !== this._valueBeforeEditing;
+        this._delegate.spreadsheetRuleHeaderFieldDidCommit(this, changed);
```

Comment:
Based on the previous condition for calling this, shouldn't we use optional chaining here since both `_delegate` as well as `spreadsheetRuleHeaderFieldDidCommit` are optional?
e.g. `this._delegate?.spreadsheetRuleHeaderFieldDidCommit?.(this, changed);`

Reply:
🤦 Indeed. I took Devin's suggestion but forgot to move the checks. Will update patch.